internal/infrastructure/kubernetes/infra.go

```diff
@@ -47,6 +47,9 @@ type Infra struct {
 	// Namespace is the Namespace used for managed infra.
 	Namespace string
```

Comment:
Hi @cnvergence do we need to change the namespace used by the ratelimit ResourceRender? It should use controllernamespace instead of namespace.

Reply:
Yep, +1 for this, clearer to understand the differences